djautotask/models.py

```diff
@@ -664,21 +664,13 @@ def get_entered_time(self):
         In Autotask, tickets are required to have start and end times.
         Start and end times for project tasks can be optional.
         In the case that a task has no start or end time, use the
-        date_worked field.
+        modified field.
         """
         if self.end_date_time:
             entered_time = self.end_date_time
         else:
-            # Autotask gives us date_worked as a datetime, even though the
-            # time is always set to EST midnight (00:00:00).
-            est_offset = timezone.localtime(
-                timezone=timezone.pytz.timezone(OFFSET_TIMEZONE)).utcoffset()
-            local_offset = timezone.localtime().utcoffset()
-
-            # We want to end up with a UTC datetime that is midnight in the
-            # local timezone.
-            date_worked = self.date_worked + est_offset
-            entered_time = date_worked - local_offset
+            # record modified time is used in the absence of end_date_time
+            entered_time = self.modified
```

Comment:
This doesn't work. The comments explain why we need to use the date_worked field. If you find yourself deleting large sections of commented code please consult myself, Cameron, or Matt. 

By changing this to date modified that will change the time entries date to anytime its data is updated, which does not reflect when a time entry was created.

Looking at the issue description it seems like the original theory for why this bug was ocurring is wrong. So I will put some time into researching the customer's problem again, and reproducing it before you continue work on this issue.